docs/types.rst

```diff
@@ -47,58 +47,34 @@ represent TEXT columns.  This was chosen for maximum forward compatibility with
 Python 3, and to make it easier to write code that will work identically in
 both languages.  This decision has many important ramifications.
```

Comment:
Let's drop the last two sentences, and just leave "This module uses byte strings to represent BLOB values, and Unicode strings to represent TEXT columns."

In the Python 3 world we certainly don't need to justify that decision, clearly Unicode strings won. 😄 

I guess we need to have some sort of sentence to explain the numbered list. Maybe something like "You may occasionally need to convert between byte strings and Unicode strings:" or something.